docs/sources/flowchart/agent-vs-direct.mmd

```diff
@@ -1,5 +1,5 @@
 flowchart TD
-    EA1(Auto-instrumenter)  --> |Push OTLP metrics & traces| GA1(Grafana Agent)
```

Comment:
I think we should rename this file too.

Reply:
I've renamed the flow chat file. What are we using to generate the flowcharts?